src/main/java/appeng/client/render/effects/LightningFX.java

```diff
@@ -33,12 +33,12 @@
 {
 
 	private static final Random RANDOM_GENERATOR = new Random();
-	final int steps = this.getSteps();
-	final double[][] Steps;
+	private final int steps = 5;
+	private final double[][] precomputedSteps;
 	private final double[] vertices = new double[3];
 	private final double[] verticesWithUV = new double[3];
-	float currentPoint = 0;
-	boolean hasData = false;
+	private final float currentPoint = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "currentPoint" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)